UnityGLTF/Assets/UnityGLTF/Runtime/Scripts/SceneImporter/ImporterTextures.cs

```diff
@@ -114,7 +114,7 @@ protected async Task ConstructImageBuffer(GLTFTexture texture, int textureIndex)
 			}
 		}
 
-		async Task CheckMimeTypeAndLoadImage(GLTFImage image, Texture2D texture, byte[] data, bool markGpuOnly)
+		async Task<Texture2D> CheckMimeTypeAndLoadImage(GLTFImage image, Texture2D texture, byte[] data, bool markGpuOnly)
```

Comment:
Can you make sure to open an issue with the KTX package that we'd a way to pass in a texture to overwrite it?
(If I understand the code here right, could also use a comment)

Reply:
My understanding is that right now there would be cases where we're leaking textures (passing one in, creating and returning a new one), is that correct?

Reply:
https://github.com/atteneder/KtxUnity/issues/85

and you're right. i will add a destroy